code/lib/preview-api/src/modules/preview-web/PreviewWeb.test.ts

```diff
@@ -180,6 +182,15 @@ describe('PreviewWeb', () => {
       expect(preview.storyStore.globals!.get()).toEqual({ a: 'c' });
     });
 
+    it('emits the PROJECT_PREPARED event', async () => {
+      await createAndRenderPreview();
+
+      expect(mockChannel.emit).toHaveBeenCalledWith(PROJECT_PREPARED, {
+        argTypes: {},
+        parameters: { docs: expect.any(Object) },
```

Comment:
should we turn this into an inline snapshot instead, or does the content change too much over time?